Makd.mak

```diff
@@ -27,7 +27,7 @@ R := $(subst $T,,$(patsubst $T/%,%,$(CURDIR)))
 PROJECT_NAME ?= $(shell basename $T)
 
 # Define the valid flavors
-VALID_FLAVORS := devel production
+VALID_FLAVORS := devel prod
```

Comment:
Maybe be nice to the user and check if the value is `production` and fail telling the user to please update it to `prod`?

Reply:
Also it should probably target v3.x.x if it is breaking...

Reply:
Ah, created the branch but didn't target it 🙈

Reply:
> Maybe be nice to the user and check if the value is production and fail telling the user to please update it to prod?

Is there any MakD user outside of the projects we maintain ?

Reply:
Not that I know of, but even if it's highly unlikely, once you head this road of "nobody is using it" is hard to make any sense to having proper versioning at all IMHO...

Reply:
For one side, you'll get an error message saying the flavor is wrong, and if you are a good library user you would have read the changelog (missing!) so it should be obvious what happened, but I still think is only 3 extra lines of code and it makes users experiences nicer if you add something like:
```make
# This should be removed in v4.x.x
ifeq ($F,production)
$(error F=$F is not a valid flavor, 'production' was renamed to 'prod', please update your code.
endif
```
after https://github.com/sociomantic-tsunami/makd/blob/v2.x.x/Makd.mak#L59 (untested).

This can be even added as a deprecation (change `error` to `warning`) to v2.x.x and support both as a migration path.

Reply:
Wouldn't those 3 lines break someone defining `production` as a valid flavor though ?

Reply:
> untested

Check your parenthesis!